src/services/services.ts

```diff
@@ -3798,6 +3798,10 @@ module ts {
                 // Try getting just type at this position and show
                 switch (node.kind) {
                     case SyntaxKind.Identifier:
+                        if(isLabelName(node)){
+                            return undefined;

```

Comment:
instead of undefined, we could return a label quick info:

```ts
return {
    kind: ScriptElementKind.label,
    kindModifiers: ScriptElementKindModifier.none,
    textSpan: createTextSpan(node.getStart(), node.getWidth()),
    displayParts: textPart((<Identifier>Node).text),
    documentation: undefined
};
```